src/app/components/products/products.component.html

```diff
@@ -1 +1,17 @@
-<p>products works!</p>
+<mat-grid-list class="product-list" cols="{{numColumns}}" [gutterSize]="'1px'">
+  <mat-grid-tile *ngFor=" let product of products">
+    <mat-card class="product-card">
+      <div class="product-image" [style.backgroundImage]="'url(' + product.image +')'"></div>
```

Comment:
can you use back-ticks here? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Unfortunately not; [reference](https://stackoverflow.com/questions/43793849/angular-4-template-binding-with-es6-template-string).  Angular has its own interpolation syntax within the HTML templates.  A more comprehensive solution would be a component like `<product-image [image]="product.image"></product-image>`.  As the app builds functionality, this might get refactored anyway.